src/layer.rs

```diff
@@ -34,10 +33,6 @@ pub struct LayerSettings {
     pub grid_size: Vec2,
     /// Size in pixels of the tilemap texture.
     pub texture_size: TextureSize,
-    /// The layer id associated with this map.
-    pub(crate) layer_id: u16,
-    /// The map id associated with this map.
-    pub(crate) map_id: u16,
```

Comment:
#139 made these public only within the crate itself. Anyway I went by removing them completely even before that PR was merged since it seemed for me to avoid potential bugs by simply completely removing these and making settings argument non mutable.